Problem1.java

```diff
@@ -0,0 +1,78 @@
+//PROBLEM - Implement Queue using 2 stacks 
+/** 3 Pointer Approcach: 
+ * 2 stacks are taken.
+ * 1st stack will be updated on each Push operation.
+ * 2nd stack willhandle all Peek and Pop operations.
+ * WhenEver a pop/peek operation is made 2nd stack will be checked -
+ * if 2nd stack is empty then all elements present in 1st stack will 
+ * be inserted to 2nd stack and then Peek or Pop us performed.
+ */ 
+//
+//time Complexity :
+// Push - Best/Amotized/Worst:O(1)
+// Pop - Best/Amotized:O(1); Worst: O(N)
+// Peek - Best/Amotized:O(1); Worst: O(N)
+
+
+// Space Complexity :
+// O(N), where N is the number of elements
+
+// Did this code successfully run on Leetcode : YES.
+// Any problem you faced while coding this : NO, However, please let me know wheather I am calculating time complexity in the right way. 
```

Comment:
You are correct with the time complexities. Pop and peek have amortized costs, but push doesn't. Only mention amortized when applicable.